src/Mod/Sketcher/Gui/TaskSketcherGeneral.ui

```diff
@@ -139,6 +139,44 @@ Points must be set closer than a fifth of the grid size to a grid line to snap.<
      </property>
     </widget>
    </item>
+   <item>
+    <widget class="Gui::PrefCheckBox" name="checkBoxAutoRemoveRedundants">
+     <property name="toolTip">
+      <string>New constraints that would be redundant will automatically be removed</string>
+     </property>
+     <property name="text">
+      <string>Auto remove redundants</string>
+     </property>
+     <property name="checked">
+      <bool>false</bool>
+     </property>
+     <property name="prefEntry" stdset="0">
+      <cstring>AutoRemoveRedundants</cstring>
+     </property>
+     <property name="prefPath" stdset="0">
+      <cstring>Mod/Sketcher</cstring>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="Gui::PrefCheckBox" name="checkBoxAutoUpdate">
+     <property name="toolTip">
+      <string>Executes a recomputation of active document after every sketch action</string>
+     </property>
+     <property name="text">
+      <string>Auto update</string>
+     </property>
+     <property name="checked">
+      <bool>false</bool>
```

Comment:
Shall be `true` to be consistent with the rest of the codebase.

Reply:
But it was false : https://github.com/FreeCAD/FreeCAD/blob/d8d9e21d238bd37a708d3ddab69730fe3f682429/src/Mod/Sketcher/Gui/TaskSketcherMessages.ui#L77
Shouldn't it be kept as it was?

Reply:
You can keep as was.